lib/inputs/pingbacks.js

```diff
@@ -56,6 +56,9 @@ module.exports = class Pingbacks {
   async ping(url, record, hostCounts) {
     try {
       await pingurl.ping(url, record, this._timeout, this._timeoutWait);
+      if (!url.match(/adzerk/)) {
```

Comment:
Still multiple adzerk domains flying around?

Reply:
we should be fine - the https change just made these go from `http://engine.adzerk.net` to `https://engine.adzerk.net`.  I can't imagine they'll move tracker pixels to the `s.zkcdn.net` domain?  That would be weird, to put it under a CDN domain.